README.md

```diff
@@ -46,12 +59,13 @@ centralized way. This should be helpful to reduce the amount of code needed to w
 operations in a pipeline for routine tasks, such as operating with containers or building container
 images.
 
-The [src/main.sh](main.sh) script is the main entrypoint and should be used to load the modules
+The [src/load_module.sh](load_module.sh) script is the main entrypoint and should be used to load
+the modules
 included in this library. This script requires all the other scripts available in a local directory
 following the same structure in this repository.
 
-To use any of the provided libraries, you must source the [src/main.sh](main.sh) script
-and pass the unique library ID to be loaded as a parameter.
+To use any of the provided libraries, you must source the [src/load_module.sh](load_module.sh)
+script and pass the unique library ID to be loaded as a parameter.
```

Comment:
Pop an example in after this line for better visualization by the reader.

Reply:
Added